build.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
 }
 
 group = "com.faire.detektrules"
-version = "0.1.0"
+version = "0.2.5"
```

Comment:
How do we have 0.2.4 released with `version = 0.1.0`?

Reply:
This should be removed.

Can you bump here: https://github.com/Faire/faire-detekt-rules/blob/main/gradle.properties

Reply:
Why do we use properties instead of the "typical" approach?

Reply:
For snapshot releases. I guess we can sed the version here instead of properties https://github.com/Faire/faire-detekt-rules/blob/main/.github/workflows/release-snapshot.yml#L27

Reply:
Why not just put it into the gradle file? `if (env.VARIABLE) version += "-snapshot"`

Reply:
Open for improvements :)

(Didn't think of that)

Reply:
Sounds good!